rollup.config.ts

```diff
@@ -103,7 +109,19 @@ const buildNode = {
 
 const buildNative = {
   input: 'src/native/index.ts',
-  external: ['events', 'node-request-interceptor'],
+  external: [
+    'tty',
```

Comment:
Not sure if it's safe to rely on these modules as external built-in modules in React Native.